core/server/master/src/main/java/alluxio/master/block/BlockMaster.java

```diff
@@ -351,4 +370,11 @@ void workerRegisterStream(
    */
   @VisibleForTesting
   long getJournaledNextContainerId();
+
+  /**
+   * The target worker must have been decommissioned.
+   * Delete target worker from decommissioned worker set.
+   * @param workerInfo the WorkerInfo of target worker.
+   */
+  void freeDecommissionedWorker(WorkerInfo workerInfo) throws NotFoundException;
```

Comment:
nit: a better naming IMO

```suggestion
  void removeDecommissionedWorker(WorkerInfo workerInfo) throws NotFoundException;
```

Reply:
Fixed.